projects/SCION/programs/basic/examples/basic.nix

```
@@ -0,0 +1,5 @@
+{ ... }:
+
+{
+  services.scion.enable = true;
```

This will try to use the service's module instead of the program's

```suggestion
  programs.scion.enable = true;
```